serverless-test-project/rule-handler.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+module.exports.handleRule = async (event) => {
+  console.log(event)
+  const e = Boolean(event.detail.triggerError)
```

Comment:
We can replace these two lines with
```
if (event.detail.triggerError) {
```
There is no need for the `Boolean(` since it should already by a boolean, or at least a truthy value.